sei/chain.json

```diff
@@ -212,13 +212,13 @@
         },
         "cosmwasm": {
           "version": "v0.0.4",
-          "repo": "https://github.com/sei-wasmd",
+          "repo": "https://github.com/sei-protocol/sei-wasmd",
           "path": "$HOME/.sei/wasm",
           "enabled": true
         },
         "ibc": {
           "type": "go",
-          "repo": "https://github.com/sei-ibc-go/v3",
+          "repo": "https://github.com/sei-protocol/sei-ibc-go/v3",
```

Comment:
We'll still need to remove the /v3 at the end of the repo property (for all cases of 'repo')